File: oa-frontend/src/types/office/PpCTaskcompletedDTO.ts
/**
 * PpCTaskcompletedDTO
 */
export interface PpCTaskcompletedDTO {
	xactivity?: string;
	xactivityAlias?: string;
	xactivityDescription?: string;
	xactivityName?: string;
	xactivityToken?: string;
	xactivityType?: string;
	xapplication?: string;
	xapplicationAlias?: string;
	xapplicationName?: string;
	xcompleted?: boolean;
	xcompletedTime?: Date;
	xcompletedTimeMonth?: string;
	xcreateTime?: Date;
	xcreatorIdentity?: string;
	xcreatorPerson?: string;
	xcreatorUnit?: string;
	xdistinguishedName?: string;
	xduration?: number;
	xempowerFromIdentity?: string;
	xexpired?: boolean;
	xid?: string;
	xidentity?: string;
	xjob?: string;
	xjoinInquire?: boolean;
	xlabel?: string;
	xlatest?: boolean;
	xopinion?: string;
	xperson?: string;
	xprocess?: string;
	xprocessAlias?: string;
	xprocessingType?: string;
	xprocessName?: string;
	xproperties?: { [key: string]: any };
	xrouteAlias?: string;
	xrouteName?: string;
	xserial?: string;
	xstartTime?: Date;
	xstartTimeMonth?: string;
	xtask?: string;
	xtitle?: string;
	xunit?: string;
	xupdateTime?: Date;
	xviewTime?: Date;
	xwork?: string;
	[property: string]: any;
}

File: oa-java/oa-domain/src/main/java/com/zeroone/star/project/dto/j5/bizcenter/PpCTaskcompletedDTO.java
package com.zeroone.star.project.dto.j5.bizcenter;

import lombok.Data;
import java.time.LocalDateTime;

/**
 * @program: oa-java
 * @description: 待办实体类DTO
 * @author: hamhuo
 * @create: 2024-10-27
 **/
@Data
public class PpCTaskcompletedDTO {
    private static final long serialVersionUID = 1L;


    private String xid;
    private String xjob;
    private String xtitle;
    private LocalDateTime xstartTime;
    private String xstartTimeMonth;
    private LocalDateTime xcompletedTime;
    private String xcompletedTimeMonth;
    private String xwork;
    private Boolean xcompleted;
    private String xapplication;
    private String xapplicationName;
    private String xapplicationAlias;
    private String xprocess;
    private String xprocessName;
    private String xprocessAlias;
    private String xserial;
    private String xperson;
    private String xidentity;
    private String xunit;
    private String xempowerFromIdentity;
    private String xactivity;
    private String xactivityName;
    private String xactivityAlias;
    private String xactivityDescription;
    private String xactivityType;
    private String xactivityToken;
    private String xcreatorPerson;
    private String xcreatorIdentity;
    private String xcreatorUnit;
    private String xrouteName;
    private String xrouteAlias;
    private String xopinion;
    private String xtask;
    private Boolean xexpired;
    private Integer xduration;
    private String xprocessingType;
    private Boolean xlatest;
    private Boolean xjoinInquire;
    private LocalDateTime xviewTime;
    private String xdistinguishedName;
    private Object xproperties;
    private String xlabel;
    private LocalDateTime xcreateTime;
    private LocalDateTime xupdateTime;

    }


